src/cmps/PokemonGrid.tsx

```diff
@@ -58,13 +60,22 @@ const PokemonGrid = ({ pokemons }: PokemonGridProps) => {
   return (
     <>
       <CardGrid spacing={2}>
-        {visiblePokemons.map((pokemon) => (
-          <PokemonCardComponent
-            key={pokemon.id}
-            pokemon={pokemon}
-            onClick={() => handleCardClick(pokemon)}
-          />
-        ))}
+        {visiblePokemons.length > 0 ? (
+          visiblePokemons.map((pokemon) => (
+            <PokemonCardComponent
+              key={pokemon.id}
+              pokemon={pokemon}
+              onClick={() => handleCardClick(pokemon)}
+            />
+          ))
+        ) : (
+          <EmptyPokemonsContainer>
+            <EmptyPokemons>
+              <img src={emptyPokemons} alt="no pokemons" />
+              No Pokemons were found.
+            </EmptyPokemons>
+          </EmptyPokemonsContainer>
```

Comment:
create an emptyState wrapper that recieve children and if the children are null|undefiend render the empty state, else the children